addons/web/static/src/legacy/js/views/calendar/calendar_controller.js

```diff
@@ -532,7 +532,15 @@ var CalendarController = AbstractController.extend({
         }).then( function () {
             self.reload();
         });
-    }
+    },
+    /**
+     * This function has been created for the only purpose of
+     * changing the title on the quick create from a calendar.
+     * This way it can be overriden in other apps
+     */
+    _setEventTitle: function () {
```

Comment:
You don't set a title, you get a title. 

The docstring is not needed. A method like _getNewEventDefaultTitle is enough.